ios/packages/swiftui/Sources/types/assets/ControlledAsset.swift

```diff
@@ -75,7 +80,7 @@ open class ControlledAsset<DataType: AssetData, ModelType>: SwiftUIAsset where M
             decoder.logger?.t("Updating model for \(data.id)")
             model.data = data
         } else {
-            self.model = ModelType(data)
+            self.model = ModelType(data, userInfo: decoder.userInfo)
```

Comment:
Do we need to update the `userInfo` when we get the model from the cache?

Reply:
good call